docs/data/routes/docs/fundamentals/cli/en.md

```diff
@@ -52,68 +110,130 @@ jss create my-jss-app nextjs --prerender SSG --fetchWith GraphQL
 jss create my-jss-app nextjs --prerender SSR --fetchWith REST
 ```
 
-### `jss setup`
+### `jss deploy app`
```

Comment:
Not sure if we have this captured anywhere else - `deploy app/files` are *not* valid for Next.js (because Next.js does not support Integrated mode, so we never need to deploy the JS files to Sitecore).

Reply:
And we come again to the framework-specific docs.